stdlib/oauth2/src/main/ballerina/src/oauth2/outbound_oauth2_provider.bal

```diff
@@ -232,11 +240,11 @@ function generateAuthTokenForOAuth2(GrantTypeConfig authConfig, @tainted Outboun
     }
 }
 
-# Process auth token for OAuth2 at inspection.
+# Process the OAuth2 token at inspection flow.
 #
 # + authConfig - OAuth2 configurations
 # + oauth2CacheEntry - OAuth2 cache entry
-# + return - Auth token or `Error` if the validation fails
+# + return - OAuth2 token or `oauth2:Error` if the validation fails
```

Comment:
```suggestion
# + return - OAuth2 token or `oauth2:Error` if the validation fails
```
Please refer the template for the correct format.

Reply:
```suggestion
# + return - OAuth2 token, or else `oauth2:Error` if the validation failed
```